Sources/docc/DocCDocumentation.docc/other-formatting-options.md

```diff
@@ -0,0 +1,108 @@
+# Other Formatting Options
+
+Improve the presentation and structure of your content by incorporating special page elements.
+
+## Overview
+
+DocC offers support for various types of elements such as tables, notes, asides, and special characters.
+
+### Add Tables of Data
+
+To create a table, start a new paragraph and use dashes (`-`) to define 
+columns, and pipes (`|`) as column separators. Construction of a table requires 
+at least three rows: 
+
+* A row that contains the column names.
+* A row with cells that contain only dashes (`-`); three or more for each 
+column.
+* One or more rows of content.
+
+```markdown
+| Sloth speed  | Description                           |
+| ------------ | ------------------------------------- | 
+| `slow`       | Moves slightly faster than a snail.   |
+| `medium`     | Moves at an average speed.            |
+| `fast`       | Moves faster than a hare.             |
+| `supersonic` | Moves faster than the speed of sound. |
+```
+There's no need to impose a column width. A column determines its width 
+using the contents of its widest cell. You can also omit the leading 
+and trailing pipes.
+
+```markdown
+Sloth speed | Description
+--- | ---
+`slow` | Moves slightly faster than a snail.
+`medium` | Moves at an average speed.
+`fast` | Moves faster than a hare.
+`supersonic` | Moves faster than the speed of sound.
+```
+
+Both examples result in the same output:
+
+| Sloth speed | Description |
+| ---: | :--- |
+| `slow` | Moves slightly faster than a snail. |
+| `medium` | Moves at an average speed. |
+| `fast` | Moves faster than a hare. |
+| `supersonic` | Moves faster than the speed of sound. |
+
+> Note: Table cells don't support multiple paragraphs, lists, or code 
```

Comment:
I think this needs to be an Important aside. The text should say something like:

You can only place a single line of text within a table cell. For example, you can't have multiple paragraphs, lists, or images within a table cell.

Reply:
Resolved https://github.com/apple/swift-docc/pull/742/commits/2e64258bcfa7eff3e8272bf1c2075245b5f4d9c5